Packs/UrlScan/Integrations/UrlScan/UrlScan.py

```diff
@@ -48,11 +48,16 @@ def http_request(method, url_suffix, json=None, wait=0, retries=0):
         headers=headers,
         verify=USE_SSL
     )
+
+    if r.headers.get('X-Rate-Limit-Remaining') == '10':
+        return_warning(f'Your rate limit has reached to 10. The rate limit will reset at '
+                       f'{r.headers.get("X-Rate-Limit-Reset")}')
```

Comment:
Maybe something like this so we can have a countdown.

```suggestion
    rate_limit_remaining = int(r.headers.get('X-Rate-Limit-Remaining'), 99)
    if int(r.headers.get('X-Rate-Limit-Remaining')) < 10:
        return_warning(f'Your available rate limit remaining is {rate_limit_remaining} and is about to be reached. The rate limit will reset at '
                       f'{r.headers.get("X-Rate-Limit-Reset")}')
```